### DISCUSSION TITLE
When will be new version?

### DISCUSSION THREAD

**Discussion by RuikangSun** [LMMS/lmms#6777] [General]
#### Justification

Unlike those unmaintained open source softwares, LMMS has an active and friendly community. There are many pull requests recently. However, lase version of LMMS is 1.2.2 which released 3 years ago. For open source developers, they can compile from source code. But for open source musicians (or just LMMS users), a software with no new release in 3 years may sound less attractive. There are a few discussions about newer version( #6573 #6604 )last year, so are our community ready for 1.3.0?

Regards.

**Comment by zonkmachine** [comment]
The short answer is the usual, it will be released when it's ready. I don't know when that is but in the mean time you have the nightly releases which I warmly recommend over the alphas. Same download location as the stable and alphas, scroll all the way down. https://lmms.io/download#linux

Please join [our discord channel](https://discord.com/invite/PruNxpG). It's more traffic there.

**Comment by LoveBodhi** [comment]
I have to wait until #4715 finish...

**Comment by sgqy** [comment]
For now I am using the nightly version.
It comes from [Actions](https://github.com/LMMS/lmms/actions/workflows/build.yml) - "first item" - Artifacts.

**Reply by Spekular** [reply]
Nightlies are also available (more conveniently) via the site: https://lmms.io/download
